src/ui/actions/session.ts

```diff
@@ -120,6 +122,7 @@ export function createSession(recordingId: string): UIThunkAction {
         recording,
         userInfo,
         auth0User: tokenManager.auth0Client?.user,
+        userSettings,
```

Comment:
Maybe just pass in `disabled: userSettings` here?

Reply:
On second thought we pass in `recording` and `userInfo` just to check its properties and see if we need to bail. Might be good to pull those out and just do the if statement here.